src/metrics/rules/active_ruleset.go

```diff
@@ -213,6 +217,27 @@ func (as *activeRuleSet) mappingsForNonRollupID(
 	}
 }
 
+func (as *activeRuleSet) LatestRollupRules(timeNanos int64) ([]view.RollupRule, error) {
```

Comment:
can we add a test?

Reply:
Done. I added a basic regression test to verify that we get back the non-tombstoned entries from the existing set of test rules.